modules/vertamediaBidAdapter.js

```diff
@@ -1,119 +1,125 @@
-import Adapter from 'src/adapter';
-import bidfactory from 'src/bidfactory';
-import bidmanager from 'src/bidmanager';
 import * as utils from 'src/utils';
-import { ajax } from 'src/ajax';
-import { STATUS } from 'src/constants';
-import adaptermanager from 'src/adaptermanager';
-
-const ENDPOINT = '//rtb.vertamedia.com/hb/';
-
-function VertamediaAdapter() {
-  const baseAdapter = new Adapter('vertamedia');
-  let bidRequest;
-
-  baseAdapter.callBids = function (bidRequests) {
-    if (!bidRequests || !bidRequests.bids || bidRequests.bids.length === 0) {
-      return;
-    }
-
-    var RTBDataParams = prepareAndSaveRTBRequestParams(bidRequests.bids[0]);
-
-    if (!RTBDataParams) {
-      return;
-    }
-
-    ajax(ENDPOINT, handleResponse, RTBDataParams, {
-      contentType: 'text/plain',
-      withCredentials: true,
-      method: 'GET'
+import {registerBidder} from 'src/adapters/bidderFactory';
+import {VIDEO} from 'src/mediaTypes';
+
+const URL = '//rtb.vertamedia.com/hb/';
+const BIDDER_CODE = 'vertamedia';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [VIDEO],
+  isBidRequestValid: function (bid) {
+    return Boolean(bid && bid.params && bid.params.aid);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests
+   * @param bidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (bidRequests, bidderRequest) {
+    return bidRequests.map((bid) => {
+      return {
+        data: prepareRTBRequestParams(bid),
+        contentType: 'text/plain',
```

Comment:
`contentType` is `text/plain` by default and `withCrendentials` is also set to true. 
Both can be removed from here.